pandas/core/resample.py

```diff
@@ -823,6 +823,14 @@ def fillna(self, method, limit: int | None = None):
         2018-01-01 01:30:00  6.0  5
         2018-01-01 02:00:00  6.0  5
         """
+        if method is not None:
```

Comment:
Unlike other fillna methods, there is no `value` argument here. But there is also a `nearest` option which you can't get with ffill/bfill. Does it make sense to add an `nfill`?

Reply:
does `interpolate` do that?

Reply:
That's a bit different; fillna does not fill existing NA values, only those created by upsampling.

```
ser = pd.Series(
    data=[1, np.nan, 3],
    index=[
        dt.datetime(2023, 3, 1, 7, 0, 0),
        dt.datetime(2023, 3, 1, 7, 0, 2),
        dt.datetime(2023, 3, 1, 7, 0, 4),
    ]
)

print(ser.resample("1s").interpolate("nearest"))
# 2023-03-01 07:00:00    1.0
# 2023-03-01 07:00:01    1.0
# 2023-03-01 07:00:02    1.0
# 2023-03-01 07:00:03    3.0
# 2023-03-01 07:00:04    3.0
# Freq: S, dtype: float64

print(ser.resample("1s").fillna(method="nearest"))
# 2023-03-01 07:00:00    1.0
# 2023-03-01 07:00:01    NaN
# 2023-03-01 07:00:02    NaN
# 2023-03-01 07:00:03    3.0
# 2023-03-01 07:00:04    3.0
# Freq: S, dtype: float64
```

This does seem to me to be useful when upsampling on a time series with irregular intervals.

Reply:
you've looked at this more closely than i have.  what's your suggestion here?

Reply:
Whoops, there is already a `.nearest`! We're good here; deprecating the entire method doesn't lose any functionality.